### ISSUE TITLE
Label attributes should be required at all times

### ISSUE THREAD

**Issue by marikaris** [molgenis/molgenis#7132]
### How to Reproduce
Upload: 
[test.xlsx](https://github.com/molgenis/molgenis/files/1685998/test.xlsx)
Go to base_table

### Expected behavior
I get my pretty table

### Observed behavior
Unknown error!
(the id attribute of ref is invisible and the label is not set)

Conclusion:
Please make the label attribute required so the importer will fail and already tell people not to do this

**Comment by bartcharbon** [comment]
Do you propose to always require a label attr, or only when the id is hidden?

This would mean a change of the EMX spec, and cause some currently valid EMX files (according to the importer) to become invalid

**Comment by Mark-de-Haan** [comment]
Fun fact: The metadata manager already forces you to select a label attribute when creating a new entity type

**Comment by mswertz** [comment]
Only when ID is hidden. It is annoying that metadata editor ask you to select label attribute when there is a perfectly find id attribute that can be shown

**Comment by dennishendriksen** [comment]
Fixed by #7424.

Observed behavior is now:
```
Error! Your import failed:
Lookup attribute [id] of entity type [base_table] must be visible
```
